LEAF_Request_Portal/sources/Form.php

```diff
@@ -2921,179 +3192,152 @@ public function query($inQuery)
-                                                                    WHERE indicatorID=:indicatorID', $tVarTypeHint);
+                        $indicatorSQL = 'SELECT format, `default` FROM indicators WHERE indicatorID=:indicatorID';
+                        $tResTypeHint = $this->db->prepared_query($indicatorSQL, $tVarTypeHint);
 
                         $vars[':indicatorID' . $count] = $q['indicatorID'];
                         $joins .= "LEFT JOIN (SELECT recordID, indicatorID, series, data FROM data
 										WHERE indicatorID=:indicatorID{$count}) lj_data{$count}
 										USING (recordID) ";
-                    }
-                    else
-                    {
-                        if ($q['indicatorID'] === '0')
-                        {
+                    } else {
+                        if ($q['indicatorID'] === '0') {
                             $joinSearchAllData = true;
-                        }
-                        else
-                        {
-                            if ($q['indicatorID'] == '0.0')
-                            { // to search all fields matching the orgchart_employee input format
+                        } else {
+                            if ($q['indicatorID'] == '0.0') { // to search all fields matching the orgchart_employee input format
                                 $joinSearchOrgchartEmployeeData = true;
                             }
                         }
                     }
 
                     // fix to select null data
-                    if ($operator == '=' && $vars[':data' . $count] == '')
-                    {
+                    if ($operator == '=' && $vars[':data' . $count] == '') {
                         $conditions .= "{$gate}(lj_data{$count}.data {$operator} :data{$count} OR lj_data{$count}.data IS NULL)";
-                    }
-                    else
-                    {
-                        if ($operator == '!=' && $vars[':data' . $count] == '')
-                        {
+                    } else {
+                        if ($operator == '!=' && $vars[':data' . $count] == '') {
                             $conditions .= "{$gate}(lj_data{$count}.data {$operator} :data{$count})";
-                        }
-                        else
-                        {
+                        } else {
                             $dataTerm = "lj_data{$count}.data";
-                            if ($joinSearchAllData
-                            || $joinSearchOrgchartEmployeeData)
-                            {
+                            if (
+                                $joinSearchAllData
+                                || $joinSearchOrgchartEmployeeData
+                            ) {
                                 $dataTerm = 'lj_data.data';
                             }
 
                             $dataMatch = ":data{$count}";
                             switch ($tResTypeHint[0]['format']) {
```

Comment:
Can you toss a toLower() on this one.  The above ones would probably take too long to fix and would require a lot of additional testing.  But it looks like this one is already using all lowercase for now.  Might as well lock it in.